code/game/machinery/newscaster.dm

```diff
@@ -877,8 +877,15 @@ var/list/obj/machinery/newscaster/allCasters = list()
 	var/creationTime
 
 /obj/item/weapon/newspaper/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] is focusing intently on [src]! It looks like they're trying to commit sudoku.</span>")
-	return(OXYLOSS)
+	user.visible_message("<span class='suicide'>[user] looks thoughtful, then triumphant, and holds the [src] aloft!</span>")
+	user.say(";JOURNALISM IS MY CALLING! EVERYBODY APPRECIATES UNBIASED REPORTI-GLORF")
+	var/mob/living/carbon/human/H = user
+	var/obj/W = new /obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(H.loc)
+	playsound(H.loc, 'sound/items/drink.ogg', rand(10,50), 1)
+	W.reagents.trans_to(H, W.reagents.total_volume)
+	user.visible_message("<span class='suicide'>[user] downs the contents of [W.name] in one gulp! They should never have picked up the [src]!</span>")
+
+	return(TOXLOSS)
```

Comment:
because @Niknakflak said he wanted a journalism-related death for a newspaper; i'm not that attached to it

and that's not magic, that's just how journalism works, don't you know anything